1.preprocess_data/scripts/0.sort_data.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
```

Comment:
Please add a markdown cell in the beginning of this notebook to describe what the notebook does.

Also, by looking at the code, it seems that `sort` in the filename is not accurate. It seems you are actually renaming/reorganizing the files. 

Naming things is one of the hardest things in computer science, but you will be more effective and other people will understand you better once you get in a good habit of accurate naming.

Reply:
Addressed by adding markdown and renaming the file!